src/services/slack.ts

```diff
@@ -22,8 +23,8 @@ function timeSafeCompare(a: string, b: string) {
 
 export interface ISlackConnector {
   validateSlackRequest(payload: any): boolean;
-  displayRepoOptions(repos: Array<string>, triggerId: string): Promise<any>;
-  parseSelection(payload: any): any;
+  displayRepoOptions(repos: Array<string>, triggerId: string, admin: boolean): Promise<any>;
+  parseSelection(payload: any, entitlement: any, repoBranchesRepository: RepoBranchesRepository): any;
```

Comment:
NIT: It looks like `entitlement` isn't used in the implementation. I think it should be `isAdmin: boolean` instead for the second parameter.

Reply:
good catch, thank you!